scripts/seedDB.js

```diff
@@ -0,0 +1,41 @@
+const mongoose = require("mongoose");
+const db = require("../models");
+mongoose.Promise = global.Promise;
+
+// This file empties the podcasts collection and inserts the podcasts below
+
+mongoose.connect(
+  process.env.MONGODB_URI || "mongodb://localhost/reactreadinglist",
```

Comment:
React Reading list huh :). I think you want to update this to be something like podcastApp or podcast-app